src/components/visualizations/utils/layout.ts

```diff
@@ -0,0 +1,17 @@
+// (C) 2020 GoodData Corporation
```

Comment:
i would move this change to `src/components/visualizations/chart/tooltip.ts`

Reply:
fixed